packages/staking/src/features/drawer/StakePoolConfirmation.tsx

```diff
@@ -321,10 +322,15 @@ export const StakePoolConfirmationFooter = ({ popupView }: StakePoolConfirmation
   } = useOutsideHandles();
   const { isBuildingTx, stakingError } = useStakingStore();
   const [isConfirmingTx, setIsConfirmingTx] = useState(false);
-  const { /* currentPortfolio,*/ portfolioMutators } = useDelegationPortfolioStore((store) => ({
+  const { currentPortfolio, portfolioMutators, draftPortfolio } = useDelegationPortfolioStore((store) => ({
     currentPortfolio: store.currentPortfolio,
+    draftPortfolio: store.draftPortfolio,
     portfolioMutators: store.mutators,
   }));
+  const [confirmationModalVisible, setConfirmationModalVisible] = useState({
+    adjustment: false,
+    reduction: false,
+  });
```

Comment:
The current name suggests it's `boolean` but it's an object with configuration.

1. What do you think about a nullable state like this? (I will follow up with another comment and show you how it simplifies things)

```suggestion
  const [openPoolManagementModal, setOpenPoolManagementModal] = useState<PoolsManagementModalType>(null);
```

2. Should we rename `PoolsManagementModal` to e.g. `PoolsManagementConfirmationModal`?

Reply:
`poolsManagementConfirmationModal` is more descriptive to me.

Reply:
Fixed on c61b8ac